apps/studio/components/interfaces/Integrations/CronJobs/CreateCronJobSheet.tsx

```diff
@@ -219,23 +226,22 @@ export const CreateCronJobSheet = ({
 
   const cronJobValues = parseCronJobCommand(selectedCronJob?.command || '', project?.ref!)
 
+  // [Joshen] Keeping the var name here generic, in case there's other cases that might fail parsing
+  const unableToParseCronJob =
```

Comment:
We should do this logic in `parseCronJobCommand` and add it to the tests. You can just do on line 116
```
if (endpoint !== '') {
  return {
        type: 'http_request',
...
}
```
and it will fall through the sql snippet case.